src/MDMMotionCurve.h

```diff
@@ -94,6 +94,11 @@ FOUNDATION_EXTERN MDMMotionCurve MDMMotionCurveMakeSpring(float mass, float tens
     NS_SWIFT_NAME(MotionCurveMakeSpring(mass:tension:friction:));
 // clang-format on
 
+// clang-format off
+FOUNDATION_EXTERN MDMMotionCurve MDMMotionCurveReversed(MDMMotionCurve timingCurve)
```

Comment:
Documentation to explain that it only works for Bezier curves?

Reply:
Done.